sourceinfo/source_code_info_test.go

```diff
@@ -154,3 +159,118 @@ func fixupProtocSourceCodeInfo(info *descriptorpb.SourceCodeInfo) {
 		}
 	}
 }
+
+func TestSourceCodeInfoExtraComments(t *testing.T) {
+	// set to true to re-generate golden output file
+	const regenerateGoldenOutputFile = false
+
+	t.Parallel()
+	compiler := protocompile.Compiler{
+		Resolver: protocompile.WithStandardImports(&protocompile.SourceResolver{
+			ImportPaths: []string{"../internal/testdata"},
+		}),
+		SourceInfoMode: protocompile.SourceInfoExtraComments,
```

Comment:
What do you think about generating a similar .txt file for standard comments so that at a glance someone could easily diff the two files to see how the different settings change the behavior?

Reply:
Done